test/benchmark/avro-upsert/README.md

```diff
@@ -5,9 +5,30 @@ assumes that Kafka topics should have 30 partitions and that you want 16 worker
 
 ## Running the Avro Upsert Test
 
-Download `genavro_400m_v2_out.gz`, unzip it, and copy it into the directory called `data` in the
-current directory. Then run:
+To run the spec benchmark, please ensure that your machine has 16 cores and at least 96GB of
+memory. Then run the following command:
 
-    ./mzcompose run test-upsert-performance
+    ./mzcompose run benchmark
 
-Navigate to `http://localhost:3000` to see the Kafka ingest performance.
+There is also a version of this benchmark that uses the same load generator, but a different set
+of views that avoid exercising the upsert operator. This gives a good comparative benchmark to see
+how much slower the upsert operator is:
+
+    ./mzcompose run benchmark-insert-filter
+
+If you're running on a smaller machine, such as a laptop with 8 cores / 32GB of memory, you can
+run the version of this benchmark designed for smaller machines:
+
+    ./mzcompose run benchmark-medium
+
+## Looking at Performance
+
+Each benchmark will output something like the following:
+
+```sh
+SUCCESS! Benchmark finished in settled 766 seconds, 522193.21148825064 rows/sec
```

Comment:
minor: It looks like the most recent version of the code says "Benchmark took <foo> seconds" but specifically "Benchmark finished in settled ..." jumped out as a small typo

Reply:
It's a typo! Will fix this up.